tests/e2e/e2e-spec.ts

```diff
@@ -33,56 +33,55 @@ describe('E2E TEST', () => {
     await server.stop();
   });
 
-  describe('Storage', () => {
-    let token: string;
-    let userId: number;
-    let rootFolderId: number;
+  const createFileOnFolder = async (folderId: number, name: string, authToken: string): Promise<request.Response> =>
+    await request(app)
+      .post('/api/storage/file')
+      .set('Authorization', `Bearer ${authToken}`)
+      .send({
+        file: {
+          fileId: uuid.v4().substring(0, 24),
+          type: 'jpg',
+          bucket: '01fa78f686158a14f8f7009b',
+          size: 57,
+          folder_id: folderId,
+          name: name,
+          encrypt_version: '03-aes',
+        },
+      });
 
-    const deleteAllUserFilesFromDatabase = async (userId: number): Promise<void> => {
-      const files = await server.models.file.findAll({ where: { user_id: userId } });
-      for (const file of files) {
-        await file.destroy();
-      }
-    };
+  const createFolder = async (body: any, authToken: string): Promise<request.Response> =>
+    await request(app).post('/api/storage/folder').set('Authorization', `Bearer ${authToken}`).send(body);
 
-    const deleteAllUserFoldersFromDatabase = async (userId: number): Promise<void> => {
-      const folders = await server.models.folder.findAll({ where: { user_id: userId } });
-      for (const dolder of folders) {
-        await dolder.destroy();
-      }
-    };
+  const deleteAllUserFilesFromDatabase = async (userId: number): Promise<void> => {
+    const files = await server.models.file.findAll({ where: { user_id: userId } });
+    for (const file of files) {
```

Comment:
Why not do a `deleteAll` by `userId`?

Reply:
`deleteAll` does not exist, but I can use delete with a query object. Will make the change